env/flink/jni_helper.cc

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2019-present, Facebook, Inc.  All rights reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#include "jni_helper.h"
+
+namespace ROCKSDB_NAMESPACE {
```

Comment:
How about changing the namespace from ROCKSDB_NAMESPACE to FORST_NAMESPACE (or FLINK_NAMESPACE) ?

Reply:
As discussed offline, we will handle this together later.